src/index.js

```diff
@@ -0,0 +1,53 @@
+import chroma from 'chroma-js';
+import def from '../default';
+import denodeify from 'promise-denodeify';
+import extend  from 'extend';
+import fs from 'fs';
+import fse from 'fs-extra';
+import { minify } from 'html-minifier';
```

Comment:
Oh, because `minify` is not the default export value of `html-minifier`.

In `require`-style, it would different in this way: `var minify = require('html-minifier').minify`.